docs/sources/whatsnew/whats-new-in-v9-5.md

```diff
@@ -0,0 +1,192 @@
+
+### Redesigned dashboard panel is generally available
+
+_Generally available in all editions of Grafana._
+
+Dashboard panels contain a lot of information, some of which is difficult to discover or access from the dashboard. With our redesigned panels, we've improved accessibility and made it easier to understand the status of a panel by adding and moving key elements.
+
+We’ve improved panels without titles, made panel descriptions and errors more succinct, and linked key actions from the header of the panel. All of these are laid out from left to right in a row, so there are no overlapping, unusable components.
+
+Grafana’s new panel design is available only for React-based panels. No Angular-based panels, like the legacy Graph and Worldmap panels, are redesigned. As a reminder, Angular is deprecated in Grafana and will be removed in a future release. See our [deprecation docs]({{< relref "../developers/angular_deprecation/" >}}) for more information.
+
+{{< figure src="/media/docs/grafana/panel-redesign-whats-new-9.5.png" max-width="750px" caption="Panel header with open menu" >}}
+
+## Prometheus performance and usability improvements
+
+### Prometheus metric encyclopedia
+
+_Experimental in all editions of Grafana._
+
+When you have thousands (or millions) of Prometheus metrics, it can be hard to find the exact one you're looking for. Enable feature toggle `prometheusMetricEncyclopedia` to replace the basic metric select dropdown in the Prometheus query builder with a paginated and searchable metric _encyclopedia_.
+
+Here's what you can do with the metric encyclopedia:
+
+- Fuzzy search for metrics by name, type, and description
+- Filter metrics by Prometheus types (gauge, counter, histogram, summary)
+- Display metrics in a paginated list, sort the results, and choose a number of results per page, so that you don't wait a long time for search results
+- View metric details, like type and description
+- [Expert feature] Search metric names by regex using the backend only
+
+### Prometheus browser cache
+
+_Experimental in all editions of Grafana._
+
+New feature toggle `prometheusResourceBrowserCache` provides the ability to cache Prometheus editor API calls in the Prometheus data source configuration.
+This improves Prometheus query editor performance, with the biggest performance improvements seen by users with high cardinality Prometheus instances.
+
+## Removal of API key creation from the UI
+
+With this update we are going one step further in deprecating API Keys in favor of [service accounts]({{< relref "../administration/service-accounts/" >}}). We've removed the button for creating new API keys through Grafana's user interface, and now only allow the creation of API keys using our HTTP API. We recommend that you migrate your existing API keys to service accounts, and opt for new service accounts instead of new API keys. This change is part of our long-term strategy for sunsetting API keys.
```

Comment:
```suggestion
With this update we are going one step further in deprecating API keys in favor of [service accounts]({{< relref "../administration/service-accounts/" >}}). We've removed the button for creating new API keys through Grafana's user interface, and now only allow the creation of API keys using our HTTP API. We recommend that you migrate your existing API keys to service accounts, and opt for new service accounts instead of new API keys. This change is part of our long-term strategy for sunsetting API keys.
```